src/clients/Lighthouse.ts

```diff
@@ -0,0 +1,32 @@
+import { AppClient, IOContext, InstanceOptions } from '@vtex/api'
+import { stringify } from 'querystring'
+
+const LH_TIMEOUT = 1e5
```

Comment:
Can you add how much is that? Is it in seconds, millisseconds? I don't think the variable must be renamed, but maybe a comment can help for better reading

Reply:
Got it!